src/Data/Lens/Iso/CoproductEither.purs

```diff
@@ -0,0 +1,11 @@
+module Data.Lens.Iso.CoproductEither where
+
+import Prelude ((<<<), flip, id)
+
+import Data.Functor.Coproduct (Coproduct(..), runCoproduct)
+import Data.Either (Either())
+
+import Data.Lens.Iso (Iso(), IsoP(), iso)
+
+_CoproductEither :: forall f g h i a. Iso (Coproduct f g a) (Coproduct h i a) (Either (f a) (g a)) (Either (h a) (i a))
```

Comment:
Ok, sorry for the nitpick, but I'd just name this `_Coproduct` to match the constructor, like we do with `Prism`s. I think that's consistent with `lens` and `derive-lenses` too. Thoughts?